SparkPost/2web2ui
# SA-2000: Add preset reports to the all reports modal

### What Changed

- Added preset report to all reports modal
- Added creator field to preset reports
- Added tests

### How To Test

- Go to /signals/analytics?
- Check that the saved reports dropdown still has preset reports with Default creator tag.
- Click `View All Reports` button
- Click the `All Reports` tab.
- Scroll down.
- Verify that the preset reports are present and you can pin/create scheduled reports from them.

- Also check that when adding/deleting a saved report, the subscription endpoint gets called and usage is updated.

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
